iozone/client.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/sh
+#
+# Environment variables used:
+#  - SERVER: hostname or IP-address of the NFS-server
+#  - EXPORT: NFS-export to test (should start with "/")
+
+# enable some more output
+set -x
+
+[ -n "${SERVER}" ]
+[ -n "${EXPORT}" ]
+
+# install build and runtime dependencies
+yum -y install git gcc nfs-utils time
+
+mkdir -p /mnt/ganesha
+
+cd ~
+curl -o iozone3_394.tar http://www.iozone.org/src/current/iozone3_394.tar
```

Comment:
nit: If possible its better to get a latest stable iozone binary available instead of sticking to iozone3_394 version.

Reply:
@soumyakoduri Sure ... will look into it .. :)

Reply:
Thanks! This can be enhanced later (as part of another pull request).

Reply:
We could put the iozone RPM in the centos-gluster*-test repository. That way you can install it with

    # yum --enablerepo=centos-gluster*test -y install iozone

Reply:
@nixpanic 
Hi, How can I add the iozone RPM to `centos-gluster*test` ?